utils/codepointtrie/builder/cpp/README.md

```diff
@@ -0,0 +1,108 @@
+# CodePointTrie Builder Tool
+
+This directory contains C++ bindings to the ICU4C CodePointTrie builder in the form of an executable named `list_to_ucptrie`. There are two ways to run this tool:
+
+1. As a WebAssembly module
+2. As a native executable
+
+The WebAssembly module is checked into tree and made available to Rust library clients.
+
+## Usage
+
+`list_to_ucptrie` takes line-separated integers in STDIN, corresponding to the trie values for code points 0, 1, 2, and so on. When it receives an EOF (`^D`), it prints helpful information to STDERR and dumps the CodePointTrie TOML to STDOUT.
+
+It takes three positional arguments:
+
+1. Default value for code points (integer)
+2. Error value for out-of-range code points (integer)
+3. TrieType: `fast` or `small`
```

Comment:
question: Should the data array value width (8-bit, 16-bit, 32-bit) be provided, too?

Reply:
I was going to infer it from `size_of::<T::ULE>`, but I forgot to add it! Thanks for catching this